src/views/Dashboard/Lab/Registration/Services/index.vue

```diff
@@ -299,46 +298,59 @@ export default {
 
     fieldRequiredRule() {
       return [
-        val => !!val || 'This field is required'
+        this.fieldRequired
       ]
     },
 
+    fieldRequired() {
+      return val => !!val || 'This field is required'
+    },
+
+    englishAlphabet() {
+      return val => (val && /^[A-Za-z0-9!@#$%^&*\\(\\)\-_=+:;"',.\\/? ]+$/.test(val)) || "This field can only contain English alphabet"
+    },
```

Comment:
Why do you separate it like this? Rather than doing this, why don't you make a separate `rules` folder inside the `common` folder instead?